### PR TITLE
[minikube] added possibility to work with minikube via kubectl

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
it is much much more clear what happens than and allows reuse kubectl command not only for minikube

### CODE REVIEW

File: content/en/docs/setup/learning-environment/minikube.md [hunk 1]
```diff
@@ -350,6 +350,12 @@ To access the [Kubernetes Dashboard](/docs/tasks/access-application-cluster/web-
 minikube dashboard
 ```
 
+Under the hood above command proxies internal cluster resources onto local host and appends dashboard API path:
```
Comment (sftim) [inline content/en/docs/setup/learning-environment/minikube.md]: @dzmitry-lahoda I'm afraid I don't understand what the reader should learn from reading this. Can you clarify?

Comment (dzmitry-lahoda) [reply content/en/docs/setup/learning-environment/minikube.md]: it is not clear why `minikube dashboard` hang after run and what it really does. But things become clear when read about `proxy`. same time user gets command which will work not only on minikube, but on production environment either (i hope).

Comment (savitharaghunathan) [reply content/en/docs/setup/learning-environment/minikube.md]: @dzmitry-lahoda, According to Style guide (https://kubernetes.io/docs/contribute/style/style-guide/#avoid-jargon-and-idioms), using a jargon is discouraged. Can you update the sentence accordingly?

**Comment by zparnold** [comment]
/close
@dzmitry-lahoda We'd love to have your contribution! Please re-open when you're ready to continue. 😄

**Pull request closed without merging**